pytorch/ITrackerData.py

```diff
@@ -267,16 +267,16 @@ def get_hog_descriptor(self, im):
     def __getitem__(self, index):
         rowIndex = self.indices[index]
         imFacePath = os.path.join(self.dataPath,
-                                  '%05d/appleFace/%05d.jpg' % (self.metadata['labelRecNum'][rowIndex],
```

Comment:
Session names and image names are no longer five digit integers.

Sessions are now a timestamp (e.g. 2020-08-12T04:49:38) and images are now target-snapshot (e.g. 00002-00013)